docs/installation/upgrade/include/2.4-converted-settings.inc

```diff
@@ -32,3 +32,17 @@
 | `quota_over_script` | Replaced by [[setting,quota_over_status]] named filter with [[setting,execute]]. |
 | `quota_max_mail_size` | Renamed to [[setting,quota_mail_size]]. The default value is set to `unlimited`. |
 | `fts_index_timeout` | Renamed to [[setting,fts_search_timeout]]. The default value is set to `30 secs`. |
+| `namespace/alias_for` | Renamed to [[setting,namespace_alias_for]]. |
+| `namespace/disabled` | Renamed to [[setting,namespace_disabled]]. |
+| `namespace/hidden` | Renamed to [[setting,namespace_hidden]]. |
```

Comment:
How about `namespace { alias_for }` syntax? That was already used for a few lines here, and I'm adding more in my other pull request.

Except, I'm thinking it's not very useful to list these namespaces ones, since 2.3 namespace configuration works for these settings just the same in 2.4.

Reply:
I am reading from this comment, that the namespace-settings change should be dropped from the 2.4-converted-settings?

Reply:
And with this the whole commit/PR?

Reply:
In my opinion yes.

Reply:
:saluting_face: